### PR TITLE
Add a .env_sample file, update gitignore, update README.md

### PR DESCRIPTION
**Pull request by thepriefy**
issue #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,24 @@ Install smtpapi-nodejs and its dependencies:
 npm install
 ```
 
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
```
Comment (mbernier) [inline README.md]: If you are doing this step, then all you need to do is open and edit the file by inserting your API KEY.

The next section `environment variable` is redundant.

**Comment by thepriefy** [comment]
I've  done what you requested. Is that ok?
(I don't know why the build fails. I there is anything I did in the wrong way, please tell me.)

**Comment by mbernier** [comment]
This is good to go, the build is failing because we added a check for files in the repo (like this one that you added). Thanks!

**Comment by thinkingserious** [comment]
Hello @thepriefy,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**